dev/cnf/resources/bnd/rejar.props

```diff
@@ -32,7 +32,7 @@ Bundle-Version: ${bFullVersion}.${if;${driver;gradle};${version.qualifier};eclip
 -consumer-policy :    ${range;[==,+)}
 
 # Require a 1.6 JDK (at least): see section 3.4.1 of OSGi 4.3 core spec
-Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.6))"
+Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.8))"
```

Comment:
This should only be incremented by 1 java version right? Java 6 --> 7.
Also, the comment on the line above needs to be updated.

Reply:
Same as above.